app/scripts/views/complete_account_unlock.js

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'views/form',
+  'views/base',
+  'stache!templates/complete_account_unlock',
+  'lib/auth-errors',
+  'lib/validate'
+],
+function (FormView, BaseView, CompleteAccountUnlockTemplate, AuthErrors,
+    Validate) {
+
+  function doesLinkValidate(uid, code, isLinkDamaged) {
+    return Validate.isUidValid(uid) &&
+           Validate.isCodeValid(code) &&
+           ! isLinkDamaged;
+  }
+
+  var CompleteAccountUnlockView = FormView.extend({
+    template: CompleteAccountUnlockTemplate,
+    className: 'complete_account_unlock',
+
+    beforeRender: function () {
+      try {
+        this.importSearchParam('uid');
```

Comment:
#2125 adds a VerificationInfoModel. One of the PRs will need to be updated. Which one depends on the merge order.